guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -670,7 +666,7 @@ that you don't already have.
 
 ### Older Versions of Ruby on Rails
 
-If you want to add a fix to older versions of Ruby on Rails, you'll need to set up and switch to your own local tracking branch. Here is an example to switch to the 4-0-stable branch:
+If you want to add a fix to older versions of Ruby on Rails, you'll need to set up and switch to your local tracking branch. Here is an example to switch to the 4-0-stable branch:
```

Comment:
I think the "own" emphasizes that the user needs to create a new branch, if they haven't already.

```suggestion
If you want to add a fix to older versions of Ruby on Rails, you'll need to set up and switch to your own local tracking branch. Here is an example to switch to the 4-0-stable branch:
```